docs/src/add-services/varnish.md

```diff
@@ -158,10 +160,8 @@ You can then access the `varnishstats` relationship over HTTP at the following p
 * `/stats`: returns the output of `varnishstat`
 * `/logs`: returns a streaming response of `varnishlog`
 
-Because of the circular relationship noted above, this can't be done on the application that Varnish is forwarding to.
-It needs to be run on a separate application container.
+To access the Varnish stats endpoint from the command line:
 
-To access the Varnish endpoint:
-- Connect to your cluster [using ssh](/development/ssh/_index.md) or through the CLI: `platform ssh -p <project id>`,
-- Display the [relationships array](../create-apps/app-reference.md#relationships) with `echo $PLATFORM_RELATIONSHIPS | base64 -d | jq '.'`,
-- Query Varnish with `curl varnishstats.internal:8081/stats`, for example, to access the statistics directly. Be sure to update the request according to the name of the relationship.
+1. Connect to your cluster [using SSH](../development/ssh/_index.md): `platform ssh -p <PROJECT_ID>`,
```

Comment:
:pick:
```suggestion
1. Connect to your cluster [using SSH](../development/ssh/_index.md): `platform ssh -p <PROJECT_ID> -A <APPNAME>`,
```

where `<APPNAME>` is the name of the second `.platform.app.yaml`, so from the previous comment that could be `varnishstats`.

Reply:
Defining the app name is probably better. But we shouldn't be using the abbreviated flags in general (though I didn't find that in the style guide quickly).